server/lib/configuration.js

```diff
@@ -494,6 +501,12 @@ var conf = module.exports = convict({
     },
     sample_rate: 1
   },
+  sync_tokenserver_url: {
+    default: 'http://127.0.0.1:5000/token',
+    doc: 'The url of the Firefox Sync tokenserver',
+    env: 'SYNC_TOKENSERVER_URL',
```

Comment:
It's also worth considering that we will change these values in production approximately never, so the probability of them being wrong in practice is very small.  Non-zero though.

@mhammond perhaps it's worth us chatting synchronously about what the expected behavior should be from the client's POV?  If you're free sometime tomorrow morning, let's chat.